contracts/crowdsale/Crowdsale.sol

```diff
@@ -110,7 +110,10 @@ contract Crowdsale {
   // -----------------------------------------
 
   /**
-   * @dev Validation of an incoming purchase. Use require statements to revert state when conditions are not met. Use super to concatenate validations.
+   * @dev Validation of an incoming purchase. Use require statements to revert state when conditions are not met. Use `super` in contracts that inherit from Crowdsale to extend their validations.
+   * @dev Example from CappedCrowdsale.sol's _preValidatePurchase method: 
```

Comment:
Nit: The second line shouldn't have a `@dev` tag. It somehow messes up Solidity's NatSpec parser. We changed this across OpenZeppelin in #995.

Reply:
Thanks! Didn't realize that. Fixed!